api/filter/accesslog/accesslog.proto

```diff
@@ -431,9 +432,8 @@ message CommonGrpcAccessLogConfig {
   // same Envoy.
   string log_name = 1 [(validate.rules).string.min_bytes = 1];
 
-  // The name of the upstream cluster that hosts the access log service. The cluster must be
-  // configured in the cluster manager.
-  string cluster_name = 2 [(validate.rules).string.min_bytes = 1];
+  // The upstream gRPC cluster that hosts the access log service.
+  GrpcCluster cluster = 2 [(validate.rules).message.required = true];
```

Comment:
As a rule of thumb, please don't reuse field number. It is best practice for using proto to avoid unpredictable data corruption.

Reply:
This feature is still under development. It's fine to reuse in this case.

Reply:
Agreed for frozen APIs. We have some detail about different API status at https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/v2_overview.html#status.

Reply:
That is fine with me. We don't allow it at Google to prevent human mistake, which can be very costly.